src/Proposal.ts

```diff
@@ -86,9 +87,10 @@ export default class Proposal {
       : this.pod.threshold;
     this.safeTransaction = safeTransaction;
     this.rejectTransaction = rejectTransaction;
+    this.status = status;
 
     if (podNonce === this.id) this.status = 'active';
-    if (podNonce > this.id) this.status = 'executed';
+    if (podNonce > this.id) this.status = 'passed';
```

Comment:
We can leave this as-is but we should probably differentiate between executed proposals that were passed vs rejected. Can you add a TODO here? I'll make a ticket once you do.